editor/src/clj/editor/disk.clj

```diff
@@ -160,18 +160,17 @@
   (let [endpoint-invalidated-since-snapshot? (g/endpoint-invalidated-pred (:snapshot-invalidate-counters post-save-actions))]
     (resource-node/merge-source-values! (:written-source-values-by-node-id post-save-actions))
     (g/cache-output-values!
-      (g/with-auto-evaluation-context evaluation-context
-        (into []
-              (keep (fn [{:keys [node-id] :as save-data}]
-                      ;; It's possible the user might have edited a resource
-                      ;; while we were saving on a background thread. We need to
-                      ;; make sure we don't add a stale save-data entry to the
-                      ;; cache.
-                      (let [save-data-endpoint (g/endpoint node-id :save-data)]
-                        (when-not (endpoint-invalidated-since-snapshot? save-data-endpoint)
-                          (pair save-data-endpoint
-                                (assoc save-data :dirty false))))))
-              (:written-save-datas post-save-actions))))
+      (into []
+            (keep (fn [{:keys [node-id] :as save-data}]
+                    ;; It's possible the user might have edited a resource
+                    ;; while we were saving on a background thread. We need to
+                    ;; make sure we don't add a stale save-data entry to the
+                    ;; cache.
+                    (let [save-data-endpoint (g/endpoint node-id :save-data)]
+                      (when-not (endpoint-invalidated-since-snapshot? save-data-endpoint)
+                        (pair save-data-endpoint
+                              (assoc save-data :dirty false))))))
+            (:written-save-datas post-save-actions)))
```

Comment:
We created an `evaluation-context` here for no reason.

Reply:
huh